Sources/SwiftCliCore/MetaDataBuilder.swift

```diff
@@ -36,24 +37,28 @@ public class MetaDataBuilder {
             if newRowMetadata.leftIncDec < 0 {
                 upcomingleftOffset += abs(newRowMetadata.leftIncDec)
                 newRowMetadata.transRowLeftOffset += upcomingleftOffset
-            }
+                }
             allRowsMetaData.append(newRowMetadata)
         }
         return(allRowsMetaData)
     }
 
     public func makeRowMetadata(stitchRow: [String], rowNumber: Int) -> RowInfo {
+
         var rowData = RowInfo(row: stitchRow, rowNumber: rowNumber)
         rowData.leftIncDec = findLeftChanges(stitchRow: stitchRow)
         rowData.rightIncDec = findRightChanges(stitchRow: stitchRow)
         rowData.width = stitchRow.count
         if rowNumber == 0 {
             rowData.bottomLine = ChartConstructor().makeBottomRow(width: rowData.width)
-        } else {
+        }
+        else
+        {
             rowData.bottomLine = ChartConstructor().makeMiddleRow(
                                                                   width: rowData.width,
                                                                   rDiff: rowData.rightIncDec,
-                                                                  lDiff: rowData.leftIncDec)
+                                                                  lDiff: rowData.leftIncDec
+                                                                  )
```

Comment:
I'm noticing is that you initiate `ChartConstructor` repeatedly here, which is a little unusual for object-oriented programming. That raises two questions for me:

- Should these functions be type methods (https://docs.swift.org/swift-book/LanguageGuide/Methods.html#ID241) rather than instance methods?
- More generally, if your class is really just a group of related functions without containing any properties, should it really be a class? Should it be a struct? Or maybe it should be a class but are there any properties that we could reference instead?

We can ask similar questions for `InputValidator`. Not a blocker for merging but something you should consider if you start to experience the limitations of this design.